[Edit] Edits to proof-by-strong-induction.md (#3237)
Codecademy/docs: content/discrete-math/concepts/proofs/terms/proof-by-strong-induction/proof-by-strong-induction.md

  - 'paths/computer-science'
---

**Proof by strong induction** is a mathematical technique for proving universal generalizations. It differs from ordinary mathematical induction (also known as weak mathematical induction) with respect to the inductive step.

In a weak mathematical induction, the inductive step involves showing that if some element _n_ has a property, then the successor element _n + 1_ must also have that property. In a strong mathematical induction, the inductive step involves showing that if all elements up to and including _n_ have some property, then _n + 1_ has that property as well.

## Steps for Strong Induction

> 1. Base Step: Prove that the first element or elements in the series have some property.
> 2. Inductive Step: Prove that if _n_ and all elements prior to n have the relevant property, then _n + 1_ has that property.

Reviewer: ```suggestion
1. `Base Step`: Prove that the first element or elements in the series have some property.
2. `Inductive Step`: Prove that if _n_ and all elements before n have the relevant property, then _n + 1_ has that property.
```
